### DISCUSSION TITLE
Release frequency

### DISCUSSION THREAD

**Discussion by Mortein** [osmlab/name-suggestion-index#5908] [Q&A] [answered]
I notice recent changes aren't visible on https://nsi.guide

Are changes generally available immediately, or is there a delay? If so, what's the frequency?
Is there is a delay for iD or other data consumers?

Thanks.

**Comment by bhousel** [comment, marked as answer]
To do a release, we need to run a script to sync our project up with Wikidata, then another script to generate the new data files, but the whole process doesn't take very long.  We typically publish releases at least once or twice a week, then they are available in iD and RapiD immediately thereafter.
